core/Tracker/VisitExcluded.php

```diff
@@ -89,8 +89,14 @@ public function isExcluded()
          * @param bool &$excluded Whether the request should be excluded or not. Initialized
          *                        to `false`. Event subscribers should set it to `true` in
          *                        order to exclude the request.
+         * @param Request $request The request object which contains all of the request's information
+         * @param bool|string $userAgent The user-agent used by the visitor
+         * @param bool|string $ip The IP-address of the visitor
+         *
+         * The additional parameters $userAgent and $ip are needed, because the values in the 
+         * object $request are empty when the constructor is called from the log-import.
```

Comment:
Sorry for that. We will likely clean up the code there and remove it from VisitExcluded shortly so it's more clear next time